docs/guide/getstarted/configuration.md

```diff
@@ -123,21 +123,22 @@ Default: *null*
 
 ## Setup [Babel](https://babeljs.io/) to write tests using next generation JavaScript
 
+**Note: these instructions are for Babel 6.  Using Babel 5 is not recommended.**
+
+First, install babel dependencies:
+```
+npm install --save-dev babel-core babel-preset-es2015
+```
+
 There are multiple ways to setup Babel using the wdio testrunner. If you are running Cucumber or Jasmine tests, you just need
 to register Babel in the before hook of your config file
 
 ```js
     before(function() {
-        require('babel-register')({
-            blacklist: [
-                'regenerator'
-            ]
-        });
+        require('babel-register');
```

Comment:
[It is](https://github.com/babel/babel/blob/master/packages/babel-core/package.json#L33), but a lot of other stuff does too that looks unnecessary?  Let me double check